nodes/core/io/21-httpin.js

```diff
@@ -232,6 +250,15 @@ module.exports = function(RED) {
                     msg.payload += chunk;
                 });
                 res.on('end',function() {
+                    if(RED.settings.metricsOn && RED.settings.metricsOn === true) {
+                        if(preRequestTimestamp) {
+                            var elapsedTimeDiff = process.hrtime(preRequestTimestamp);
+                            node.metric(msg, "request.duration.nanos", (elapsedTimeDiff[0] * 1e9 + elapsedTimeDiff[1]));
```

Comment:
Are we leaving the http request node's code as is written in this pull request?

I've yet to convert one or the other to millis/nanos, I hope I won't forget to do this.